multisig_1_of_n_account/README.md

```diff
@@ -0,0 +1,127 @@
+# Multisig 1-of-n Contract Account
+
+This example contains a custom contract account that authorizes when one ed25519
+signature is provided, where the signature was produced by a ed25519 key
+configured in the contract. The contract may hold any number of keys, and any
+key may authorize for it.
+
+The example also contains a stellar-cli plugin that signs authorizations using
+an ed25519.
+
+## Usage
+
+The example below sets up an asset with the contract account as the admin. The
+admin authorizes with ed25519 keys.
+
+The ed25519 keys used in the example below are:
+
+- Secret Key:
+  `0000000000000000000000000000000000000000000000000000000000000000`\
+  Public Key: `3b6a27bcceb6a42d62a3a8d02a6f0d73653215771de243a63ac048a18b59da29`
+- Secret Key:
+  `0000000000000000000000000000000000000000000000000000000000000001`\
+  Public Key: `4cb5abf6ad79fbf5abbccafcc269d85cd2651ed4b885b5869f241aedf0a5ba29`
+
+These keys are publicly viewable and not random. Do not use these keys for any
+purpose. Select your own keys and update them in the commands below when
+executing.
+
+### Install the `stellar sign-auth-ed25519` plugin
+
+Install one of the stellar sign-auth-ed25519 plugin implementations.
+
+#### Rust
+
+```
+cd stellar-cli-sign-auth-ed25519
+cargo install --locked --path .
+```
+
+#### JavaScript (Deno)
+
+```
+cd stellar-cli-sign-auth-ed25519-js
+deno install \
+    --global \
+    --name stellar-sign-auth-ed25519 \
```

Comment:
Maybe we should use the `make install` command? At the very least the `--name` should be `stellar-sign-auth-ed25519-js` or else it will conflict with the Rust plugin name.

Reply:
```
% ls ~/.deno/bin
deno				deno.lock			stellar-sign-auth-ed25519-js			stellar-sign-auth-ed25519
```

Reply:
d60ba6b